win32ss/drivers/font/ftfd/freetype.rc

```diff
@@ -1,8 +1,8 @@
 #define REACTOS_VERSION_DLL
-#define REACTOS_STR_FILE_DESCRIPTION   "FreeType font handling library"
-#define REACTOS_STR_INTERNAL_NAME      "freetype"
-#define REACTOS_STR_ORIGINAL_FILENAME  "freetype.dll"
-#define REACTOS_STR_COMPANY_NAME       "ReactOS Development Team/FreeType Team"
-#define REACTOS_STR_LEGAL_COPYRIGHT    "Copyright 1998-2011 ReactOS Team"
-#define REACTOS_STR_ORIGINAL_COPYRIGHT "Copyright 1996-2013 FreeType Team"
+#define REACTOS_STR_FILE_DESCRIPTION   "FreeType font handling library\0"
+#define REACTOS_STR_INTERNAL_NAME      "freetype\0"
+#define REACTOS_STR_ORIGINAL_FILENAME  "freetype.dll\0"
+#define REACTOS_STR_COMPANY_NAME       "ReactOS Development Team/FreeType Team\0"
+#define REACTOS_STR_LEGAL_COPYRIGHT    "Copyright 1998-2025 ReactOS Team\0"
+#define REACTOS_STR_ORIGINAL_COPYRIGHT "Copyright 1996-2019 FreeType Team\0"
```

Comment:
Are these `\0` required?

Reply:
Yes. This is string for VersionInfo.

https://learn.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource

Reply:
No, the \0 are NOT required, please revert.

Reply:
```suggestion
#define REACTOS_STR_FILE_DESCRIPTION   "FreeType font handling library"
#define REACTOS_STR_INTERNAL_NAME      "freetype"
#define REACTOS_STR_ORIGINAL_FILENAME  "freetype.dll"
#define REACTOS_STR_COMPANY_NAME       "ReactOS Development Team/FreeType Team"
#define REACTOS_STR_LEGAL_COPYRIGHT    "Copyright 1998-2025 ReactOS Team"
#define REACTOS_STR_ORIGINAL_COPYRIGHT "Copyright 1996-2019 FreeType Team"
```

Reply:
> https://learn.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource

Yeah, I see 2 `\0` in the example (only),
yet, it seems they are useless and should not be there...